internal/compiler/generator/rust.rs

```diff
@@ -311,6 +311,7 @@ fn generate_public_component(llr: &llr::PublicComponent) -> TokenStream {
             }
         }
 
+        #[allow(dead_code)] // FIXME: some global are unused because of optimization, we should then remove them completely
```

Comment:
I was about to ask - this pass leaves some properties unused now and I'm wondering what happens to them heh

Reply:
As a TODO:
 1. make another pass that counts the use of each property
 2. further inline the properties that are only used once
 4. optimize (do not generate) the property that are not used at all.
 5. optimize (do not generate) the global components that have no properties as a result